pkg/operator/kube_cloud_config/controller_test.go

```
@@ -252,3 +258,129 @@ SubnetID = subnet-test
+			if err := fake.Tracker().Add(&corev1.ConfigMap{
+				ObjectMeta: metav1.ObjectMeta{Name: "kube-cloud-config", Namespace: "openshift-config-managed"},
+			}); err != nil {
+				t.Fatalf("failed to add kube-cloud-config ConfigMap to fake tracker: %v", err)
+			}
+
+			indexerInfra := cache.NewIndexer(cache.MetaNamespaceKeyFunc, cache.Indexers{})
+			if err := indexerInfra.Add(inputInfra); err != nil {
+				t.Fatal(err.Error())
+			}
+
+			fakeConfig := configfakeclient.NewSimpleClientset(inputInfra)
```

golangci-lint (2.11.4)

[error] SA1019: configfakeclient.NewSimpleClientset is deprecated: NewClientset replaces this with support for field management, which significantly improves server side apply testing. NewClientset is only available when apply configurations are generated (e.g. via --with-applyconfig). 

(staticcheck)